packages/fast-usdc-contract/src/exos/operator-kit.ts

```diff
@@ -1,43 +1,45 @@
+import { OperatorKitI } from '@agoric/fast-usdc/src/operator-kit-interface.js';
 import { makeTracer } from '@agoric/internal';
+import type { Zone } from '@agoric/zone';
 import { Fail } from '@endo/errors';
-import { OperatorKitI } from '@agoric/fast-usdc/src/operator-kit-interface.js';
-
-const trace = makeTracer('TxOperator');
+import type {
+  CctpTxEvidence,
+  RiskAssessment,
+} from '@agoric/fast-usdc/src/types.js';
 
-/**
- * @import {Zone} from '@agoric/zone';
- * @import {CctpTxEvidence, RiskAssessment} from '@agoric/fast-usdc/src/types.js';
- */
+const trace: (message: string) => void = makeTracer('TxOperator');
```

Comment:
I guess this type suffices for the usage in this file, but I'd expect:

```suggestion
const trace: (...args: unknown[]) => void = makeTracer('TxOperator');
```